pkg/apis/v1/nodeclaim_status.go

```diff
@@ -27,6 +27,7 @@ const (
 	ConditionTypeInitialized          = "Initialized"
 	ConditionTypeEmpty                = "Empty"
 	ConditionTypeDrifted              = "Drifted"
+	ConditionTypeUnderutilized        = "Underutilized"
```

Comment:
Did we think about if representing the status condition as "Underutilized" is the right name here? Thinking more on this -- if a user is setting this value to 0 to get their old behavior, they are going to have all of their nodes marked as underutilized which I could see being confusing -- I wonder if we considered other names here